### PR TITLE
Editor: Display again mute button for non-critical quality checks

### LINKED ISSUE #4098: Bring back mute button for non-critical quality checks
This request from Rhos, Welsh localisater on mlo _"Any chance of getting the mute button back on the Quality Checks? I prefer to be able to clear the deck so that I can see where I am and what's new."_

In losing the mute button for non-critical checks we made it less noisy and reduced the need to mute the non-critical checks.  But we also made it impossible for a translator to mark all false positives and address all issues in their translations.  Thus they can't actually perform a clean review of a particular issue.

I'd suggest we bring back the mute button for this instance, but ideally hide it for these tests unless the user hovers over the item.

### PR DESCRIPTION
**Pull request by unho**
Fixes #4098.

### CODE REVIEW

**Comment by julen** [comment]
Tested, and the text jumps when hovering over the check name. Let's fix that.

**Comment by unho** [comment]
I am sorry, but I haven't seen any vertical jump.

What I see is that if you hover a failing check, thus displaying the mute button, the next check(s) sometimes jump to the left or the right. But I think that is ok.

**Comment by unho** [comment]
@translate/core If anyone else can please test this will be much appreciated. Make sure to check out the branch and rebuild the assets, then find some unit with several non-critical failing checks.

**Comment by julen** [comment]
This is in Firefox
checks

**Comment by unho** [comment]
Thanks for the screenshot, that explains the issue you are experiencing pretty well. I am still unable to replicate this vertical jump.

**Comment by unho** [comment]
@julen I think the vertical jumping is fixed. I would appreciate if you can quickly test it again.

File: pootle/templates/editor/units/xhr_checks.html [hunk 1]
```diff
@@ -9,8 +9,8 @@
       <a href="{% url 'pootle-checks-descriptions' %}#{{ check.name }}" target="_blank">{{ check.display_name }}</a>
       {% if canreview %}
       <a class="js-toggle-check toggle-check" data-check-id="{{ check.id }}">
-	<i title="{% trans 'Mute quality check' %}" class="mute-check icon-block"></i>
-	<i title="{% trans 'Unmute quality check' %}" class="unmute-check icon-block-muted"></i>
+        <i title="{% trans 'Mute quality check' %}" class="mute-check icon-block"></i>
```
Comment (julen) [inline pootle/templates/editor/units/xhr_checks.html]: This cleanup doesn't belong here, feel free to do it directly out of this PR.

**Comment by julen** [comment]
I still experience the vertical jump. This fixes it https://github.com/julen/pootle/commit/a8968710998349a5e11555e9395748d1c4f9815e

File: pootle/templates/editor/units/xhr_checks.html (hunk 1 above)
Comment (unho) [inline pootle/templates/editor/units/xhr_checks.html]: Will do.

**Pull request merged**